src/storage-client/src/controller/rehydration.rs

```diff
@@ -150,21 +154,21 @@ struct RehydrationTask<T> {
     metrics: RehydratingStorageClientMetrics,
 }
 
-enum RehydrationTaskState {
+enum RehydrationTaskState<T: Timestamp + Lattice> {
     /// Wait for the address of the storage replica to connect to.
     AwaitAddress,
     /// The storage replica should be (re)hydrated.
     Rehydrate {
         /// The network address of the storage replica.
```

Comment:
nit: rustdoc could be updated here, and below. But it's not super important for now.